content/components/banner.mdx

```diff
@@ -21,45 +21,30 @@ Banner can be used in one of two ways:
 
 ### Feedback Banners
 
-Banners that are used to communicate feedback require extra accessibility considerations to ensure they are surfaced immediately to all users including those using assistive technologies. This relates to [WCAG 4.1.3 Status Messages](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html).
+Banners that are used to communicate feedback require extra accessibility considerations to ensure they are immediately perceived by assistive technology users.
 
-Example scenarios that require extra considerations include:
+Consider using either a **live region announcement** or **focus management** technique:
 
-* When a user submits a form, an error banner is shown with a list of form errors that need to be corrected.
-* When a user submits a form, the submission successfully goes through and a success banner is shown.
-* When a user saves a setting, an error banner is shown to communicate that an unexpected error has occured and the setting was not saved.
-* When a user toggles an option, a warning banner is shown to warn the user of potential consequences.
-
-Banners used in these scenarios can be made more accessible through either a **focus management** or a **live region announcement** technique:
-
-* **Focus management** would involve moving focus to the newly shown Banner. This is the most reliable way to ensure that a user is aware of the new content. This is the preferred method for critical Banner scenarios (with exceptions).
-* **Live region announcements**: A live region announcement can be used to announce the new content to screen reader users. This method is less reliable than focus management, but may be the preferred method for non-critical information (with exceptions).
+* **Live region announcements**: can be used to announce the new content to screen reader users. This method may be the preferred method for non-critical information (**with exceptions**).
+* **Focus management**: involves shifting a user's focus directly the new Banner. This method is disruptive when used inappropriately, but is extremely helpful in scenarios where we need to guide the user towards an action (**with exceptions**).
```

Comment:
A link to the exceptions or an indication that these exceptions will be mentioned later in the doc would be helpful.